src/jsutils/instanceOf.ts

```diff
@@ -1,10 +1,12 @@
 import { inspect } from './inspect.js';
 
 /* c8 ignore next 3 */
-const isDevelopment =
-  globalThis.process != null &&
+const isProduction =
+  globalThis.process == null ||
```

Comment:
Most bundlers won't eliminate the || you can run tests at https://github.com/JoviDeCroock/graphql-minifier-experiments